WooCommerce/Classes/ViewRelated/Orders/Order Creation/ProductsSection/ConfigurableBundleItemViewModel.swift

```diff
@@ -268,35 +268,39 @@ private extension ConfigurableBundleItemViewModel {
         let minQuantityString = NumberFormatter.localizedString(from: bundleItem.minQuantity as NSDecimalNumber, number: .decimal)
         if let maxQuantity = bundleItem.maxQuantity {
             let maxQuantityString = NumberFormatter.localizedString(from: maxQuantity as NSDecimalNumber, number: .decimal)
-            return String(format: Localization.ErrorMessage.invalidQuantityWithMinAndMaxQuantityRules, minQuantityString, maxQuantityString)
+            return String(format: Localization.ErrorMessage.invalidQuantityWithMinAndMaxQuantityRulesFormat, product.name, minQuantityString, maxQuantityString)
         } else {
-            return String(format: Localization.ErrorMessage.invalidQuantityWithMinQuantityRule, minQuantityString)
+            return String(format: Localization.ErrorMessage.invalidQuantityWithMinQuantityRuleFormat, product.name, minQuantityString)
         }
     }
 }
 
 private extension ConfigurableBundleItemViewModel {
     enum Localization {
         enum ErrorMessage {
-            static let invalidQuantityWithMinQuantityRule = NSLocalizedString(
-                "configureBundleItemError.invalidQuantityWithMinQuantityRule",
-                value: "The quantity needs to be at least %1$@",
-                comment: "Error message when setting a bundle item's quantity with a minimum quantity rule."
+            static let invalidQuantityWithMinQuantityRuleFormat = NSLocalizedString(
+                "configureBundleItemError.invalidQuantityWithMinQuantityRuleFormat",
+                value: "The quantity of %1$@ needs to be at least %2$@.",
+                comment: "Error message when setting a bundle item's quantity with a minimum quantity rule. " +
+                "%1$@ is the variable product name. %2$@ is the minimum quantity."
```

Comment:
super nit: This applies to all products, not just variable products:

```suggestion
                "%1$@ is the product name. %2$@ is the minimum quantity."
```

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/11306/commits/d0d6dcfbdd2e8d94bc0283004a37df8540d13d00